.github/workflows/ci.yml

```diff
@@ -398,21 +398,47 @@ jobs:
         sudo cat /proc/sys/vm/mmap_rnd_bits
         sudo sysctl -w vm.mmap_rnd_bits=28
     - name: config
-      run: CC=clang ./config --banner=Configured no-fips --strict-warnings -fsanitize=thread && perl configdata.pm --dump
+      run: CC=clang ./config --banner=Configured no-shared no-fips --strict-warnings -g -fsanitize=thread && perl configdata.pm --dump
```

Comment:
Why is `no-shared` necessary here?

Reply:
That's a long story: https://github.com/openssl/openssl/pull/26815#issuecomment-2665541075
without that code any stack trace is completely unreadable, because unfortunately clang uses a "symbolizer" that crashes because it links to the libcrypto.so.3 that we are testing.

I had to do the same to the memory sanitizer build 01e657c5468a637959395b07b385e58785c486f2 also with mixed feelings.

Reply:
If you think that this looses significant test coverage, I could try to get this test run use gcc with thread sanitizer instead, but I would have to add a tsan bug, to verify that it works also in the CI environment.

Reply:
OK then.